gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "13.1.3",
+  "version": "13.1.4",
```

Comment:
Given that this is for IQ, you should probably base it off the `12.x-backports` branch and target that branch with the PR.

Reply:
> Given that this is for IQ, you should probably base it off the `12.x-backports` branch and target that branch with the PR.

So I should merge [this PR](https://github.com/sonatype/sonatype-react-shared-components/pull/1176) first since it's based off `12.x-backports`. After that, create another PR to merge `12.x-backports` into `main`?